flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/utils/PlannerMocks.java

```diff
@@ -69,8 +77,25 @@ public static FlinkPlannerImpl createDefaultPlanner() {
                         functionCatalog.asLookup(parser::parseIdentifier),
                         catalogManager.getDataTypeFactory(),
                         parser::parseSqlExpression));
+    }
+
+    public FlinkPlannerImpl getPlanner() {
         return planner;
     }
 
-    private PlannerMocks() {}
+    public ParserImpl getParser() {
+        return parser;
+    }
+
+    public CatalogManager getCatalogManager() {
+        return catalogManager;
+    }
+
+    public Catalog getCurrentCatalog() {
```

Comment:
That's a shorthand for testing purpose, in order to avoid cluttering the test code with the "details" of the `CatalogManager` usage. Does it makes sense?